components/plan-build/bin/shared.bash

```diff
@@ -90,7 +90,7 @@ _render_metadata_FILES() {
   # shellcheck disable=2154
   find "$pkg_prefix" -type f \
     | sort \
-    | hab pkg hash > "${pkg_name}"_blake2bsums
+    | $HAB_BIN pkg hash > "${pkg_name}"_blake2bsums
```

Comment:
I know it seems unlikely that value will contain spaces or globbing characters, but it's not impossible, so this would be safer as
```suggestion
    | "$HAB_BIN" pkg hash > "${pkg_name}"_blake2bsums
```

Reply:
Yeah, for sure, but then it will also be inconsistent with all of our other uses of `$HAB_BIN`.  I could change all of them to `"$HAB_BIN"` but tbph, I'm terrified as it is of touching any of this code.

Reply:
I don't see a downside to quoting `"$HAB_BIN"` everywhere. If there's code that depends on it expanding globs or something, we should probably learn about that because 🤦‍♂️